### DISCUSSION TITLE
Show Payment Method By Store(s)

### DISCUSSION THREAD

**Discussion by MatthewKennedy** [spree/spree#10682] [Ideas]
Currently the options for showing a payment method in a store are:
- Chose one store.
- Leave blank to show in all stores.

Would it be better to have a tag system so that you can tag multiple  stores (A, B miss C and tag store D). You could still use the if null show in all stores.

Any thoughts on this?

**Comment by damianlegawiec** [comment]
Yes, definitely swapping this for one to many associations and allowing the store owner to choose which stores should use this method
